src/octokit/get-octokit-throttle-options.ts

```diff
@@ -1,6 +1,12 @@
 import Bottleneck from "bottleneck";
 import Redis from "ioredis";
 import { Logger } from "pino";
+import { throttling } from "@octokit/plugin-throttling";
+
+type ThrottlingOptions = Exclude<
+  Parameters<typeof throttling>[1]["throttle"],
+  undefined
+>;
```

Comment:
We could export the type directly in `@octokit/plugin-throttling` to avoid doing this

Reply:
Done in https://github.com/octokit/plugin-throttling.js/pull/631, released in `@octokit/plugin-throttling` v8.0.1

Reply:
thx

Reply:
also updated this PR